main/ethernet.c

```diff
@@ -222,6 +222,13 @@ void ethernet_init()
     esp_eth_phy_t * phy = esp_eth_phy_new_lan8720 (&phy_config);
     esp_eth_config_t config = ETH_DEFAULT_CONFIG (mac, phy);
     esp_eth_handle_t eth_handle = NULL;
-    ESP_ERROR_CHECK (esp_eth_driver_install (&config, &eth_handle));
-    ESP_ERROR_CHECK (esp_eth_start (eth_handle));
+    esp_err_t err_code = esp_eth_driver_install (&config, &eth_handle);
+    if(ESP_OK == err_code)
+    {
+      esp_eth_start (eth_handle);
```

Comment:
The returned error code is ignored. Maybe it's worth writing a message to the log?

Reply:
The following lines are 
```
    }
    else
    {
         ESP_LOGE (TAG, "Ethernet fail");
    }
```
Ideally this error would be stored somehow and printed out to factory testing system